apps/src/codebridge/Editor/Editor.tsx

```diff
@@ -58,7 +60,7 @@ export const Editor = ({langMapping, editableFileTypes}: EditorProps) => {
       {file ? (
         <CodeEditor
           key={`${file.id}/${1}`}
-          darkMode={true}
+          darkMode={theme === 'Dark'}
```

Comment:
Do we need `darkMode` prop? Could we use `useTheme` in `CodeEditor`?

Reply:
good catch!

Reply:
updated!